pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "Todo"
+version = "0.1.0"
+description = "Todo app"
+authors = ["Your Name <you@example.com>"]
+[tool.poetry.dependencies]
+python = "^3.8.5"
+certifi="2020.6.20"
+chardet="3.0.4"
+click="7.1.2"
+Flask="1.1.1"
+idna="2.10"
+itsdangerous="1.1.0"
+Jinja2="2.11.2"
+MarkupSafe="1.1.1"
+oauthlib="3.1.0"
+py-trello="0.16.0"
+python-dateutil="2.8.1"
+python-dotenv="0.12.0"
+pytz="2020.1"
+requests="2.24.0"
+requests-oauthlib="1.3.0"
+six="1.15.0"
+urllib3="1.25.9"
+Werkzeug="1.0.1"
```

Comment:
This far longer than necessary. You do not need to include nested dependencies so this can be shrunk to something more like
```
python = "^3.8.5"
flask = "^1.1.1"
python-dotenv = "^0.14.0"
requests = "^2.24.0"
python-dateutil="2.8.1"
```

It will be updated automatically in future when you add new dependencies via `poetry add`